### DISCUSSION TITLE
HYPERPARAMETERS FOR TRAINING YOLOV8 ON VISDRONE DATASET

### DISCUSSION THREAD

**Discussion by abhinav70291** [ultralytics/ultralytics#1594] [Q&A]
Hello everyone.I am currently working on detecting objects using aerial imagery such as Visdrone dataset.however using the default hyperparamter settings as used by ultralytics, I trained the yolov8n model for  around 27 epochs and the mAP 0.5 0.95  score saturates at around 0.05 . Any help in determining the hyperparameter tuning for training on Visdrone would be appreciated. Also does anyone have pretrained weights for visdone?

**Comment by glenn-jocher** [comment]
@abhinav70291 we don't have pretrained VisDone weights nor hyps, but your numbers seem very low, I would train longer.

**Reply by glenn-jocher** [reply]
@abhinav70291 in my example training I see 0.08 mAP50-95 after just 2 epochs, so I'm not able to reproduce your training results. 

<img width="1112" alt="Screenshot 2023-03-24 at 16 05 58" src="https://user-images.githubusercontent.com/26833433/227563528-b6025f1e-252f-4f15-8dd6-c65b9abd19d5.png">

**Reply by abhinav70291** [reply]
May I know which pretrained model was your base model?I chose yolov8s.pt

**Reply by glenn-jocher** [reply]
yolov8n.pt

**Reply by abhinav70291** [reply]
Thanks.it's working fine

**Reply by glenn-jocher** [reply]
@abhinav70291 great to hear that! If you have any more questions or need further assistance, feel free to ask. Happy training! 🚀
